tabular/src/autogluon/tabular/predictor/predictor.py

```diff
@@ -942,6 +948,20 @@ def fit(
         verbosity = kwargs.get("verbosity", self.verbosity)
         set_logger_verbosity(verbosity)
 
+        if verbosity >= 2:
+            if verbosity == 2:
+                logger.log(20, f"Verbosity: 2 (Standard Logging)")
+            elif verbosity == 3:
+                logger.log(20, f"Verbosity: 3 (Detailed Logging)")
+            elif verbosity >= 4:
+                logger.log(20, f"Verbosity: {verbosity} (Maximum Logging)")
+
+        include_gpu_count = False
+        if verbosity >= 3:
+            include_gpu_count = True
```

Comment:
nit: this can be done in a more Pythonic way

```python
include_gpu_count = True if verbosity >= 3 else False
```

Reply:
Good point! Updated to

```python3
include_gpu_count = (verbosity >= 3)
```